projects/batfish/src/org/batfish/grammar/cisco/CiscoParser.g4

```diff
@@ -1,6 +1,6 @@
 parser grammar CiscoParser;
 
-import
```

Comment:
In the future, please run autoformat on grammar files before committing. Unfortunately it is not done as part of project cleanup; you must autoformat each file from eclipse manually with the shortcut.